### ISSUE TITLE
Entire vendor build fails silently when an addon uses app.import to import a non-existent file

### ISSUE THREAD

**Issue by davewasmer** [ember-cli/ember-cli#2877]
Just spent a while figuring this one out, I'm not sure of the root cause.

Looks like if an addon attempts to use `app.import` in it's `included` hook, and the file it tries to import doesn't exist, the entire vendor file fails to build:

```js
// my-addon/index.js
module.exports = {
  name: 'my-addon',
  included: function(app) {
    app.import('bower_components/module/some-file-that-doesnt-actually-exist.js');
  }
};
```
Results in this `vendor.js` file in the `dist/assets/` folder:
```js
//# sourceMappingURL=vendor.map
```

No errors in the terminal output, no indication that anything went wrong, other than vendor being empty. I can try creating a test case repo if that helps.

**Comment by ChristianGreinke** [comment]
Found the same. Some third party lib breaks the build. But as it failes silently i can't not figure out exactly where. I assume it is ember-validations. Tried both 1.0. and 2.0.0.-alpha.1

**Comment by stefanpenner** [comment]
Seems like a regression

**Comment by rahult** [comment]
Found the same, forgot to run ```ember generate ember-cli-simple-auth```

However no errors in the terminal output, no indication that anything went wrong, other than vendor being empty

**Comment by ef4** [comment]
This was a bug in broccoli-sourcemap-concat, it's fixed in 0.4.3. If you reinstall all node modules you should pick up the fix.

**Comment by sir-dunxalot** [comment]
Incase it helps anyone else, I've had this issue appear up after a dependency was uninstalled but an import to one of the dependency's files was left in the Brocfile. The import wasn't throwing a node fs error but removing it corrected the vendor.js file content.
